### DISCUSSION TITLE
can you use adonis without typescript?

### DISCUSSION THREAD

**Discussion by dionsaur84** [adonisjs/core#4024] [Help]
i dont like typescript

can i use adonis without it or nah?

**Comment by Barbapapazes** [comment]
You can't. Buy because Typescript is just a superset, you can write plain javascript.

**Comment by mkevison** [comment]
When I started I had this same thought about typescript, but now I wonder how I managed to work with pure javascript.

Today I'm studying mobile development with Kotlin and I already think the typescript is a little weak in terms of typing, such as typing for numbers, flot, int, etc...
